code/datums/diseases/advance/symptoms/cough.dm

```diff
@@ -78,6 +78,8 @@ BONUS
 				addtimer(CALLBACK(M, /mob/.proc/emote, "cough"), 6)
 				addtimer(CALLBACK(M, /mob/.proc/emote, "cough"), 12)
 				addtimer(CALLBACK(M, /mob/.proc/emote, "cough"), 18)
-			if(infective && M.CanSpreadAirborneDisease())
-				A.spread(1)
+			if(infective && !(A.spread_flags & DISEASE_SPREAD_FALTERED) && prob(50))
+				addtimer(CALLBACK(A, /datum/disease/.proc/spread, 2), 20)
```

Comment:
Why does it spread 2 seconds after they cough?

Reply:
so people have a second to back off